index.html

```diff
@@ -259,6 +259,15 @@ <h6 class="description">Hello I’m a software developer! I can help you build a
       </section>
     </section>
   </div>
-       </section>
+</section>
+<footer class="form-cont">
+  <form action="https://formspree.io/f/mrgdkyla" method="post">
+    <h2 class="h2">I'm always<br>interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <input type="text" id="name" placeholder="Full name" required maxlength="30"><div class="line"></div>
+    <input type="email" placeholder="Email address" required><div class="line"></div>
+    <textarea name="" maxlength="500" required>Enter text here</textarea>
+    <button type="submit" class="get-in see1">Get in touch</button>
+  </form>
+</footer>
```

Comment:
- [ ] I kindly suggest that you add the social media icons and horizontal line from the Figma design.

| Your Design  | Figma Design |
| ------------- | ------------- |
| <img width="249" alt="image" src="https://user-images.githubusercontent.com/52934128/193899052-32c52267-5093-49a5-9b0a-f39c008a9b72.png">  | <img width="188" alt="image" src="https://user-images.githubusercontent.com/52934128/193898699-9fed16e7-139d-4e78-8e8f-74ea22a6c4bf.png">  |